### DISCUSSION TITLE
Create GitHub Releases for new releases

### DISCUSSION THREAD

**Discussion by rusty-snake** [rust-lang/rust-bindgen#2517] [General]
Tools installed via `cargo install` have one major drawback, update checks. Unlike other tools like `dnf`, `flatpak` or even `rustup` there is no simple `(check-)update` command that checks/updates all installed packages. One way to be sure you get updates for `cargo install`ed programs is to watch the repository for releases which work great for all programs I've installed via `cargo install`. Except for `bindgen(-cli)`.

So I and a lot more people I guess would be happy if you could create a GitHub Release for new versions in addition to the git-tag.

**Comment by pvdrz** [comment]
Sure this shouldn't be a problem. 

I added an extra step to the `CONTRIBUTING.md` file documenting this: https://github.com/rust-lang/rust-bindgen/pull/2519

**Reply by rusty-snake** [reply]
Thanks
